sonar-scanner-engine/src/main/java/org/sonar/scanner/issue/ignore/EnforceIssuesFilter.java

```diff
@@ -73,6 +79,7 @@ public boolean accept(FilterableIssue issue, IssueFilterChain chain) {
 
   @CheckForNull
   private String getRelativePath(String componentKey) {
+    // TODO extract componentStore which is immutable
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)